.github/workflows/ci.yml

```diff
@@ -159,7 +178,26 @@ jobs:
           --health-retries 5
         ports:
           - 5433:5432
+    env:
+      MINIO_ACCESS_KEY: minioadmin
+      MINIO_SECRET_KEY: minioadmin
+      AWS_ACCESS_KEY_ID: minioadmin
+      AWS_SECRET_ACCESS_KEY: minioadmin
+      # https://github.com/aws/aws-cli/issues/5262#issuecomment-705832151
+      AWS_EC2_METADATA_DISABLED: true
     steps:
+      # GitHub Services doesn't let us forward a command when starting the docker container
+      # and we need to run minio like this `docker start minio/minio server /data`.
+      # And, as of today, there are no easy way to pass the `server /data` part.
+      # Therefore, we start minio in docker manually
+      - name: Setup storage with minio
+        run: |
+          docker run -d \
+                     --name minio \
+                     -p 9000:9000 \
+                     -e MINIO_ROOT_USER=$MINIO_ACCESS_KEY \
+                     -e MINIO_ROOT_PASSWORD=$MINIO_SECRET_KEY \
+                     minio/minio server /data
```

Comment:
I don't really know the implications of not being able to use Github Services, but if that's an issue, couldn't we thinly wrap this image with one applying those hardcoded run arguments?

Reply:
I developed a bit on this point in the PR description. 
To be honest, I don't think there are huge differences when using this approach.
Minio did that [here](https://github.com/minio/minio/blob/master/Dockerfile.cicd). But they don't host it on docker hub ... We could host it on our side. Be it means we would have to maintain it up to date. 
And again, I don't know if it would change anything for GitHub actions

Reply:
Yes okay that's what I thought too!